### ISSUE TITLE
Provide Azure Data Studio vsix as release asset

### ISSUE THREAD

**Issue by pascalberger** [PowerShell/vscode-powershell#4488]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.

### Summary

I'm packaging the Azure Data Studio PowerShell extension for Chocolatey. For keeping the package up to date I'm checking releases on this repository. While the extension seems to be built from this repository it seems to be a different vsix than for Visual Studio Code, also with a different ID.

### Proposed Design

Make the Azure Data Studio vsix available as asset in the specific releases

**Comment by andschwa** [comment]
We don't create the ADS VSIX, they repackage that themselves. Perhaps @Charles-Gagnon can help you?

**Comment by Charles-Gagnon** [comment]
We don't repackage anything - we just upload the VS Code VSIX to our own "marketplace". So the VSIX attached to the releases here are the same ones as used in ADS and should function there as well - although keep in mind that new releases here may not always work in ADS as we are many months behind VS Code so newer extensibility features may not be available in ADS until we're able to catch up.

**Comment by andschwa** [comment]
Hmm...if that's the case, then that's what we've been doing since I took over maintaining stuff circa 2020...Can I delete this file? [PowerShell for Azure Data Studio README](https://github.com/PowerShell/vscode-powershell/blob/main/docs/azure_data_studio/README_FOR_MARKETPLACE.md)? Because _we're_ not creating a separate VSIX with this as the readme, and if you aren't either, then it's just old cruft.

And now I wonder why @pascalberger is seeing a "different" VSIX and ID in your gallery.

**Comment by pascalberger** [comment]
Now I'm confused :) 

If I install the VSIX from the release page of this repository in through the CLI in Azure Data Studio, I get the same README and ID as the extension has when installed in Visual Studio.

When I install the extension through the UI in Azure Data Studio I get a different ID and README, but you say the it is the same VSIX. 

Shouldn't installing the same VSIX through the CLI lead to the same result as installing through the UI?

**Comment by Charles-Gagnon** [comment]
That's just a bit of a gap in the ADS experience currently.

In the marketplace when you don't have the extension installed it will use the link I provided above from the gallery, which in this case is the ADS-specific README.

But as soon as you install an extension it will use the README from the VSIX itself, which means that unless we make a separate VSIX it will switch to the "VS Code" one. 

Before installation:

![image](https://user-images.githubusercontent.com/28519865/229623588-99e3ff09-5dc3-4855-9a1c-e5f3bd045694.png)

After installation (and reload)

![image](https://user-images.githubusercontent.com/28519865/229623761-e098eb84-8341-4f7a-b65d-7481289c1832.png)

I've filed an issue to track fixing this (https://github.com/microsoft/azuredatastudio/issues/22595) but just note that it's purely a display issue and the content from the VSIX is all the same so there should be no functionality difference.
